src/pyscipopt/scip.pyx

```diff
@@ -535,6 +624,19 @@ cdef class Node:
         """Retrieve the estimated value of the best feasible solution in subtree of the node"""
         return SCIPnodeGetEstimate(self.scip_node)
 
+    def getAddedConss(self):
```

Comment:
Do you know why getNAddedConss() always returns zero when calling self.getModel().getNAddedConss() inside a branch rule? Consequently, I empirically always see getAddedConss() return an empty list. This happens even when I see the total number of constraints of the model change.